.wordlist.txt

```diff
@@ -4,15 +4,18 @@ APIs
 balancer
 Charmhub
 CLI
+cron
```

Comment:
Could you please add these words to .custom_wordlist.txt? .wordlist.txt is now reserved for common usage across teams and we aren't supposed to modify it. (The separation helps when we need to update the common framework/style related files)